interface/wx/uilocale.h

```diff
@@ -374,10 +374,11 @@ class wxLocaleIdent
 
         The following tag syntax is accepted:
 
-        - BCP-47:  \<language\>[-\<script\>][-\<region\>][-\<extension\>]
-        - Windows: \<language\>[-\<script\>][-\<region\>][-\<extension\>][_\<sortorder\>]
-        - POSIX:   \<language\>[_\<region\>][.\<charset\>][@@\<modifier\>]
-        - macOS:   \<language\>[-\<script\>][_\<region\>]
+        - BCP-47:      \<language\>[-\<script\>][-\<region\>][-\<extension\>]
+        - Windows:     \<language\>[-\<script\>][-\<region\>][-\<extension\>][_\<sortorder\>]
+        - Windows CRT: \<language\>[_\<region\>][.\<charset\>]
```

Comment:
This seems to be a strict substring of the POSIX format just below, do we need to document it separately? I think it could be more useful to just write that region names are OS-dependent.

If we do document it, we should probably call it "MSVC" as other Windows compilers are perfectly free to use different formats.

Reply:
> This seems to be a strict substring of the POSIX format just below, do we need to document it separately? I think it could be more useful to just write that region names are OS-dependent.

The structure is the same, but "language" and "region" do not conform to ISO 639 resp ISO 3166 as is the case for POSIX. "language" and "region" are plain English names (at least I hope they are restricted to English). Not sure whether we need to document this here.

> If we do document it, we should probably call it "MSVC" as other Windows compilers are perfectly free to use different formats.

IMHO we should document it, because it's really not the same type of tag, even if the structure is the same. We can call it MSVC, although I find this too restrictive, because MinGW uses the same Microsoft runtime library. I don't know whether other compilers bring their own implementation.